plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java

```diff
@@ -109,40 +109,49 @@ protected Builder()
             super(createSession(Optional.of(new SelectedRole(ROLE, Optional.of("admin")))));
         }
 
-        public Builder setSkipTimezoneSetup(boolean skipTimezoneSetup)
+        public B setSkipTimezoneSetup(boolean skipTimezoneSetup)
         {
             this.skipTimezoneSetup = skipTimezoneSetup;
-            return this;
+            return (B) this;
         }
 
-        public Builder setHiveProperties(Map<String, String> hiveProperties)
+        public B setHiveProperties(Map<String, String> hiveProperties)
         {
             this.hiveProperties = ImmutableMap.copyOf(requireNonNull(hiveProperties, "hiveProperties is null"));
-            return this;
+            return (B) this;
         }
 
-        public Builder setInitialTables(Iterable<TpchTable<?>> initialTables)
+        public B addHiveProperty(String key, String value)
+        {
+            this.hiveProperties = ImmutableMap.<String, String>builder()
+                    .putAll(hiveProperties)
+                    .put(key, value)
```

Comment:
if we don't envision overrides, make `hiveProperties` an `ImmutableMap.Builder`

else, go via `HashMap` here.